tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -374,3 +382,131 @@ def bgp_title_pattern_munge(tests, id, provider=nil)
   t[:vrf] = 'default' if t[:vrf].nil?
   t
 end
+
+# setup_mt_full_env
+# Check and set up prerequisites for Multi-Tenancy Full (MT-full) testing.
+# MT-full currently requires an F3 line module. This method will update
+# the tests hash with the names of the default vdc and also an appropriate
+# test interface name to use for testing.
+# tests[:vdc] The default vdc name
+# tests[:intf] A compatible interface to use for MT-full testing.
+def setup_mt_full_env(tests, testcase)
+  # MT-full tests require a specific linecard. Search for a compatible
+  # module and enable it.
+
+  testheader = tests[:testheader]
```

Comment:
Should this be an optional parameter just in case someone does not want to pass this information in?  `testheader = tests[:testheader] unless tests[:testheader].nil?`